2023-08-21 08:17:38,500 INFO    StreamThr :3009662 [internal.py:wandb_internal():86] W&B internal server running at pid: 3009662, started at: 2023-08-21 08:17:38.499155
2023-08-21 08:17:38,503 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: status
2023-08-21 08:17:38,506 INFO    WriterThread:3009662 [datastore.py:open_for_write():85] open: /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/run-d8rz8lfi.wandb
2023-08-21 08:17:38,511 DEBUG   SenderThread:3009662 [sender.py:send():369] send: header
2023-08-21 08:17:38,562 DEBUG   SenderThread:3009662 [sender.py:send():369] send: run
2023-08-21 08:17:43,562 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: keepalive
2023-08-21 08:17:48,564 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: keepalive
2023-08-21 08:17:53,565 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: keepalive
2023-08-21 08:17:58,567 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: keepalive
2023-08-21 08:18:03,569 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: keepalive
2023-08-21 08:18:08,570 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: keepalive
2023-08-21 08:18:11,000 INFO    SenderThread:3009662 [dir_watcher.py:__init__():211] watching files in: /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files
2023-08-21 08:18:11,000 INFO    SenderThread:3009662 [sender.py:_start_run_threads():1100] run started: d8rz8lfi with start time 1692605858.494744
2023-08-21 08:18:11,001 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: status_report
2023-08-21 08:18:11,004 DEBUG   SenderThread:3009662 [sender.py:send_request():396] send_request: summary_record
2023-08-21 08:18:11,006 INFO    SenderThread:3009662 [sender.py:_save_file():1354] saving file wandb-summary.json with policy end
2023-08-21 08:18:11,014 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: check_version
2023-08-21 08:18:11,015 DEBUG   SenderThread:3009662 [sender.py:send_request():396] send_request: check_version
2023-08-21 08:18:12,006 INFO    Thread-12 :3009662 [dir_watcher.py:_on_file_created():272] file/dir created: /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files/wandb-summary.json
2023-08-21 08:18:13,769 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: run_start
2023-08-21 08:18:13,809 DEBUG   HandlerThread:3009662 [system_info.py:__init__():31] System info init
2023-08-21 08:18:13,810 DEBUG   HandlerThread:3009662 [system_info.py:__init__():46] System info init done
2023-08-21 08:18:13,810 INFO    HandlerThread:3009662 [system_monitor.py:start():181] Starting system monitor
2023-08-21 08:18:13,810 INFO    SystemMonitor:3009662 [system_monitor.py:_start():145] Starting system asset monitoring threads
2023-08-21 08:18:13,811 INFO    HandlerThread:3009662 [system_monitor.py:probe():201] Collecting system info
2023-08-21 08:18:13,812 INFO    SystemMonitor:3009662 [interfaces.py:start():190] Started cpu monitoring
2023-08-21 08:18:13,813 INFO    SystemMonitor:3009662 [interfaces.py:start():190] Started disk monitoring
2023-08-21 08:18:13,815 INFO    SystemMonitor:3009662 [interfaces.py:start():190] Started gpu monitoring
2023-08-21 08:18:13,816 INFO    SystemMonitor:3009662 [interfaces.py:start():190] Started memory monitoring
2023-08-21 08:18:13,822 INFO    SystemMonitor:3009662 [interfaces.py:start():190] Started network monitoring
2023-08-21 08:18:13,862 DEBUG   HandlerThread:3009662 [system_info.py:probe():195] Probing system
2023-08-21 08:18:13,873 DEBUG   HandlerThread:3009662 [system_info.py:_probe_git():180] Probing git
2023-08-21 08:18:13,891 DEBUG   HandlerThread:3009662 [system_info.py:_probe_git():188] Probing git done
2023-08-21 08:18:13,891 DEBUG   HandlerThread:3009662 [system_info.py:probe():240] Probing system done
2023-08-21 08:18:13,892 DEBUG   HandlerThread:3009662 [system_monitor.py:probe():210] {'os': 'Linux-5.4.0-156-generic-x86_64-with-glibc2.10', 'python': '3.8.0', 'heartbeatAt': '2023-08-21T08:18:13.862649', 'startedAt': '2023-08-21T08:17:38.479630', 'docker': None, 'cuda': '10.2.89', 'args': ('config/train_gpt2.py',), 'state': 'running', 'program': 'train.py', 'codePath': 'train.py', 'git': {'remote': 'git@github.com:kkkjj-d/nanoGPT-eva.git', 'commit': 'eba36e84649f3c6d840a93092cb779a260544d08'}, 'email': '1262670315@qq.com', 'root': '/home/yxdong/nano-gpt-test/nanoGPT-eva', 'host': 'gpu16', 'username': 'yxdong', 'executable': '/home/yxdong/anaconda3/envs/dyx-py38/bin/python', 'cpu_count': 40, 'cpu_count_logical': 80, 'cpu_freq': {'current': 1028.602175, 'min': 800.0, 'max': 3900.0}, 'cpu_freq_per_core': [{'current': 800.0, 'min': 800.0, 'max': 3900.0}, {'current': 800.112, 'min': 800.0, 'max': 3900.0}, {'current': 800.838, 'min': 800.0, 'max': 3900.0}, {'current': 887.138, 'min': 800.0, 'max': 3900.0}, {'current': 800.464, 'min': 800.0, 'max': 3900.0}, {'current': 801.001, 'min': 800.0, 'max': 3900.0}, {'current': 800.337, 'min': 800.0, 'max': 3900.0}, {'current': 841.059, 'min': 800.0, 'max': 3900.0}, {'current': 814.278, 'min': 800.0, 'max': 3900.0}, {'current': 800.876, 'min': 800.0, 'max': 3900.0}, {'current': 801.073, 'min': 800.0, 'max': 3900.0}, {'current': 800.588, 'min': 800.0, 'max': 3900.0}, {'current': 799.916, 'min': 800.0, 'max': 3900.0}, {'current': 800.384, 'min': 800.0, 'max': 3900.0}, {'current': 800.739, 'min': 800.0, 'max': 3900.0}, {'current': 800.557, 'min': 800.0, 'max': 3900.0}, {'current': 800.708, 'min': 800.0, 'max': 3900.0}, {'current': 800.897, 'min': 800.0, 'max': 3900.0}, {'current': 3001.661, 'min': 800.0, 'max': 3900.0}, {'current': 3004.128, 'min': 800.0, 'max': 3900.0}, {'current': 3658.166, 'min': 800.0, 'max': 3900.0}, {'current': 800.12, 'min': 800.0, 'max': 3900.0}, {'current': 800.102, 'min': 800.0, 'max': 3900.0}, {'current': 800.319, 'min': 800.0, 'max': 3900.0}, {'current': 800.597, 'min': 800.0, 'max': 3900.0}, {'current': 800.502, 'min': 800.0, 'max': 3900.0}, {'current': 801.053, 'min': 800.0, 'max': 3900.0}, {'current': 2072.677, 'min': 800.0, 'max': 3900.0}, {'current': 800.155, 'min': 800.0, 'max': 3900.0}, {'current': 976.189, 'min': 800.0, 'max': 3900.0}, {'current': 800.482, 'min': 800.0, 'max': 3900.0}, {'current': 800.002, 'min': 800.0, 'max': 3900.0}, {'current': 800.292, 'min': 800.0, 'max': 3900.0}, {'current': 799.915, 'min': 800.0, 'max': 3900.0}, {'current': 800.249, 'min': 800.0, 'max': 3900.0}, {'current': 800.326, 'min': 800.0, 'max': 3900.0}, {'current': 801.138, 'min': 800.0, 'max': 3900.0}, {'current': 800.103, 'min': 800.0, 'max': 3900.0}, {'current': 800.249, 'min': 800.0, 'max': 3900.0}, {'current': 800.966, 'min': 800.0, 'max': 3900.0}, {'current': 801.117, 'min': 800.0, 'max': 3900.0}, {'current': 800.736, 'min': 800.0, 'max': 3900.0}, {'current': 800.638, 'min': 800.0, 'max': 3900.0}, {'current': 806.413, 'min': 800.0, 'max': 3900.0}, {'current': 800.778, 'min': 800.0, 'max': 3900.0}, {'current': 800.635, 'min': 800.0, 'max': 3900.0}, {'current': 799.943, 'min': 800.0, 'max': 3900.0}, {'current': 886.801, 'min': 800.0, 'max': 3900.0}, {'current': 884.894, 'min': 800.0, 'max': 3900.0}, {'current': 799.897, 'min': 800.0, 'max': 3900.0}, {'current': 800.137, 'min': 800.0, 'max': 3900.0}, {'current': 800.678, 'min': 800.0, 'max': 3900.0}, {'current': 800.406, 'min': 800.0, 'max': 3900.0}, {'current': 800.281, 'min': 800.0, 'max': 3900.0}, {'current': 800.522, 'min': 800.0, 'max': 3900.0}, {'current': 800.726, 'min': 800.0, 'max': 3900.0}, {'current': 800.924, 'min': 800.0, 'max': 3900.0}, {'current': 800.259, 'min': 800.0, 'max': 3900.0}, {'current': 3625.233, 'min': 800.0, 'max': 3900.0}, {'current': 3627.741, 'min': 800.0, 'max': 3900.0}, {'current': 3526.98, 'min': 800.0, 'max': 3900.0}, {'current': 800.578, 'min': 800.0, 'max': 3900.0}, {'current': 800.545, 'min': 800.0, 'max': 3900.0}, {'current': 800.206, 'min': 800.0, 'max': 3900.0}, {'current': 800.94, 'min': 800.0, 'max': 3900.0}, {'current': 801.124, 'min': 800.0, 'max': 3900.0}, {'current': 800.602, 'min': 800.0, 'max': 3900.0}, {'current': 2099.217, 'min': 800.0, 'max': 3900.0}, {'current': 800.732, 'min': 800.0, 'max': 3900.0}, {'current': 907.769, 'min': 800.0, 'max': 3900.0}, {'current': 800.361, 'min': 800.0, 'max': 3900.0}, {'current': 1017.239, 'min': 800.0, 'max': 3900.0}, {'current': 800.557, 'min': 800.0, 'max': 3900.0}, {'current': 801.056, 'min': 800.0, 'max': 3900.0}, {'current': 800.617, 'min': 800.0, 'max': 3900.0}, {'current': 800.437, 'min': 800.0, 'max': 3900.0}, {'current': 801.033, 'min': 800.0, 'max': 3900.0}, {'current': 800.226, 'min': 800.0, 'max': 3900.0}, {'current': 800.419, 'min': 800.0, 'max': 3900.0}, {'current': 800.557, 'min': 800.0, 'max': 3900.0}], 'disk': {'total': 686.5967483520508, 'used': 167.91360473632812}, 'gpu': 'NVIDIA GeForce RTX 2080 Ti', 'gpu_count': 4, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 2080 Ti', 'memory_total': 11811160064}, {'name': 'NVIDIA GeForce RTX 2080 Ti', 'memory_total': 11811160064}, {'name': 'NVIDIA GeForce RTX 2080 Ti', 'memory_total': 11811160064}, {'name': 'NVIDIA GeForce RTX 2080 Ti', 'memory_total': 11811160064}], 'memory': {'total': 503.28995513916016}}
2023-08-21 08:18:13,892 INFO    HandlerThread:3009662 [system_monitor.py:probe():211] Finished collecting system info
2023-08-21 08:18:13,892 INFO    HandlerThread:3009662 [system_monitor.py:probe():214] Publishing system info
2023-08-21 08:18:13,892 DEBUG   HandlerThread:3009662 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-08-21 08:18:13,894 DEBUG   HandlerThread:3009662 [system_info.py:_save_pip():67] Saving pip packages done
2023-08-21 08:18:13,894 DEBUG   HandlerThread:3009662 [system_info.py:_save_conda():74] Saving list of conda packages installed into the current environment
2023-08-21 08:18:14,005 INFO    Thread-12 :3009662 [dir_watcher.py:_on_file_created():272] file/dir created: /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files/requirements.txt
2023-08-21 08:18:14,006 INFO    Thread-12 :3009662 [dir_watcher.py:_on_file_created():272] file/dir created: /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files/conda-environment.yaml
2023-08-21 08:18:16,575 DEBUG   HandlerThread:3009662 [system_info.py:_save_conda():86] Saving conda packages done
2023-08-21 08:18:16,580 INFO    HandlerThread:3009662 [system_monitor.py:probe():216] Finished publishing system info
2023-08-21 08:18:16,588 DEBUG   SenderThread:3009662 [sender.py:send():369] send: files
2023-08-21 08:18:16,588 INFO    SenderThread:3009662 [sender.py:_save_file():1354] saving file wandb-metadata.json with policy now
2023-08-21 08:18:16,590 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: status_report
2023-08-21 08:18:16,603 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: stop_status
2023-08-21 08:18:16,604 DEBUG   SenderThread:3009662 [sender.py:send_request():396] send_request: stop_status
2023-08-21 08:18:17,009 INFO    Thread-12 :3009662 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files/conda-environment.yaml
2023-08-21 08:18:17,009 INFO    Thread-12 :3009662 [dir_watcher.py:_on_file_created():272] file/dir created: /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files/wandb-metadata.json
2023-08-21 08:18:17,704 INFO    wandb-upload_0:3009662 [upload_job.py:push():133] Uploaded file /tmp/tmpwfffvjyowandb/kfgeiipx-wandb-metadata.json
2023-08-21 08:18:20,822 DEBUG   SenderThread:3009662 [sender.py:send():369] send: telemetry
2023-08-21 08:18:21,824 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: status_report
2023-08-21 08:18:26,825 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: status_report
2023-08-21 08:18:31,604 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: stop_status
2023-08-21 08:18:31,605 DEBUG   SenderThread:3009662 [sender.py:send_request():396] send_request: stop_status
2023-08-21 08:18:36,605 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: keepalive
2023-08-21 08:18:41,606 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: keepalive
2023-08-21 08:18:44,835 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: partial_history
2023-08-21 08:18:47,246 DEBUG   HandlerThread:3009662 [handler.py:handle_request():144] handle_request: poll_exit
2023-08-21 08:18:47,902 INFO    cpu       :3009662 [interfaces.py:monitor():140] Process cpu has exited.
2023-08-21 08:18:47,904 DEBUG   SystemMonitor:3009662 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2023-08-21 08:18:47,904 DEBUG   SystemMonitor:3009662 [system_monitor.py:_start():166] Finished system metrics aggregation loop
2023-08-21 08:18:47,904 DEBUG   SystemMonitor:3009662 [system_monitor.py:_start():170] Publishing last batch of metrics
2023-08-21 08:18:51,586 WARNING StreamThr :3009662 [internal.py:is_dead():414] Internal process exiting, parent pid 3009581 disappeared
2023-08-21 08:18:51,586 ERROR   StreamThr :3009662 [internal.py:wandb_internal():152] Internal process shutdown.
2023-08-21 08:18:51,910 INFO    HandlerThread:3009662 [handler.py:finish():854] shutting down handler
2023-08-21 08:18:51,910 INFO    HandlerThread:3009662 [system_monitor.py:finish():190] Stopping system monitor
2023-08-21 08:18:51,910 INFO    HandlerThread:3009662 [interfaces.py:finish():202] Joined cpu monitor
2023-08-21 08:18:51,911 INFO    HandlerThread:3009662 [interfaces.py:finish():202] Joined disk monitor
2023-08-21 08:18:51,911 INFO    HandlerThread:3009662 [interfaces.py:finish():202] Joined gpu monitor
2023-08-21 08:18:51,911 INFO    WriterThread:3009662 [datastore.py:close():298] close: /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/run-d8rz8lfi.wandb
2023-08-21 08:18:51,912 INFO    HandlerThread:3009662 [interfaces.py:finish():202] Joined memory monitor
2023-08-21 08:18:51,912 INFO    HandlerThread:3009662 [interfaces.py:finish():202] Joined network monitor
2023-08-21 08:19:16,098 INFO    SenderThread:3009662 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/urllib3/connection.py", line 200, in _new_conn
    sock = connection.create_connection(
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/urllib3/util/connection.py", line 85, in create_connection
    raise err
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/urllib3/util/connection.py", line 73, in create_connection
    sock.connect(sa)
socket.timeout: timed out

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/urllib3/connectionpool.py", line 790, in urlopen
    response = self._make_request(
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/urllib3/connectionpool.py", line 491, in _make_request
    raise new_e
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/urllib3/connectionpool.py", line 467, in _make_request
    self._validate_conn(conn)
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1092, in _validate_conn
    conn.connect()
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/urllib3/connection.py", line 604, in connect
    self.sock = sock = self._new_conn()
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/urllib3/connection.py", line 209, in _new_conn
    raise ConnectTimeoutError(
urllib3.exceptions.ConnectTimeoutError: (<urllib3.connection.HTTPSConnection object at 0x7fd840057040>, 'Connection to api.wandb.ai timed out. (connect timeout=10)')

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/urllib3/connectionpool.py", line 844, in urlopen
    retries = retries.increment(
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/urllib3/util/retry.py", line 515, in increment
    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fd840057040>, 'Connection to api.wandb.ai timed out. (connect timeout=10)'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 320, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/wandb/sdk/lib/gql_request.py", line 55, in execute
    request = self.session.post(self.url, **post_args)
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/requests/sessions.py", line 637, in post
    return self.request("POST", url, data=data, json=json, **kwargs)
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/yxdong/anaconda3/envs/dyx-py38/lib/python3.8/site-packages/requests/adapters.py", line 507, in send
    raise ConnectTimeout(e, request=request)
requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fd840057040>, 'Connection to api.wandb.ai timed out. (connect timeout=10)'))
2023-08-21 08:19:29,812 INFO    SenderThread:3009662 [sender.py:finish():1526] shutting down sender
2023-08-21 08:19:29,813 INFO    SenderThread:3009662 [dir_watcher.py:finish():359] shutting down directory watcher
2023-08-21 08:19:30,028 INFO    SenderThread:3009662 [dir_watcher.py:finish():389] scan: /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files
2023-08-21 08:19:30,029 INFO    SenderThread:3009662 [dir_watcher.py:finish():403] scan save: /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files/config.yaml config.yaml
2023-08-21 08:19:30,029 INFO    SenderThread:3009662 [dir_watcher.py:finish():403] scan save: /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files/wandb-summary.json wandb-summary.json
2023-08-21 08:19:30,030 INFO    SenderThread:3009662 [dir_watcher.py:finish():403] scan save: /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files/requirements.txt requirements.txt
2023-08-21 08:19:30,044 INFO    SenderThread:3009662 [dir_watcher.py:finish():403] scan save: /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files/conda-environment.yaml conda-environment.yaml
2023-08-21 08:19:30,051 INFO    SenderThread:3009662 [dir_watcher.py:finish():403] scan save: /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files/wandb-metadata.json wandb-metadata.json
2023-08-21 08:19:30,051 INFO    SenderThread:3009662 [file_pusher.py:finish():159] shutting down file pusher
2023-08-21 08:19:30,052 INFO    SenderThread:3009662 [file_pusher.py:join():164] waiting for file pusher
2023-08-21 08:19:30,700 INFO    wandb-upload_0:3009662 [upload_job.py:push():133] Uploaded file /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files/config.yaml
2023-08-21 08:19:31,109 INFO    wandb-upload_1:3009662 [upload_job.py:push():133] Uploaded file /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files/wandb-summary.json
2023-08-21 08:19:32,260 INFO    wandb-upload_2:3009662 [upload_job.py:push():133] Uploaded file /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files/requirements.txt
2023-08-21 08:19:36,866 INFO    wandb-upload_3:3009662 [upload_job.py:push():133] Uploaded file /home/yxdong/nano-gpt-test/nanoGPT-eva/wandb/run-20230821_081738-d8rz8lfi/files/conda-environment.yaml
